src/OpenTelemetry.AutoInstrumentation/Util/PropertyFetcher.cs

```diff
@@ -1,4 +1,18 @@
-// From https://github.com/dotnet/runtime/blob/master/src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs
```

Comment:
This comment seems worth to preserve below the header.

Reply:
25234cb2e130d72103d1ff0f4330ce36d1f2f1bc